demo/kserve/install-manual.md

```diff
@@ -0,0 +1,133 @@
+# Manual Installation
+## Export Environment Variables
+Set the variables below with desired values.
+`TARGET_OPERATOR` could be either `odh` or `rhods`.
+~~~
+export TARGET_OPERATOR=odh
+~~~
+## Clone the repo and setup the environment
+~~~
+git clone https://github.com/opendatahub-io/caikit-tgis-serving
+cd caikit-tgis-serving/demo/kserve
+
+source ./scripts/env.sh
+export TARGET_OPERATOR_TYPE=$(getOpType $TARGET_OPERATOR)
+export TARGET_OPERATOR_NS=$(getOpNS)
+export KSERVE_OPERATOR_NS=$(getKserveNS)
+~~~
+
+## Install Service Mesh operators
+~~~
+oc apply -f custom-manifests/service-mesh/operators.yaml
+sleep 30
+oc wait --for=condition=ready pod -l name=istio-operator -n openshift-operators --timeout=300s
+oc wait --for=condition=ready pod -l name=jaeger-operator -n openshift-operators --timeout=300s
+oc wait --for=condition=ready pod -l name=kiali-operator -n openshift-operators --timeout=300s
+~~~
+
+## Create an Istio instance
+~~~
+oc create ns istio-system
+oc apply -f custom-manifests/service-mesh/smcp.yaml
+sleep 30
+oc wait --for=condition=ready pod -l app=istiod -n istio-system --timeout=300s
+oc wait --for=condition=ready pod -l app=istio-ingressgateway -n istio-system --timeout=300s
+oc wait --for=condition=ready pod -l app=istio-egressgateway -n istio-system --timeout=300s
+oc wait --for=condition=ready pod -l app=jaeger -n istio-system --timeout=300s
+~~~
+
+## Install Knative Serving
+~~~
+oc create ns ${KSERVE_OPERATOR_NS}
+oc create ns knative-serving
+oc -n istio-system apply -f custom-manifests/service-mesh/smmr-${TARGET_OPERATOR_TYPE}.yaml 
+oc apply -f custom-manifests/service-mesh/peer-authentication.yaml
+oc apply -f custom-manifests/service-mesh/peer-authentication-${TARGET_OPERATOR_TYPE}.yaml 
+~~~
+This is needed because of [this](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/serverless/serving#serverless-domain-mapping-custom-tls-cert_domain-mapping-custom-tls-cert).
```

Comment:
Nit: 
`We use PeerAuthentications to enable mTLS according to [Openshift Serverless Documentation](https://access.redhat.com/documentation/en-us/red_hat_openshift_serverless/1.28/html/serving/configuring-custom-domains-for-knative-services#serverless-domain-mapping-custom-tls-cert_domain-mapping-custom-tls-cert)`